core/src/main/kotlin/gropius/dto/input/architecture/UpdateIMSInput.kt

```diff
@@ -2,22 +2,33 @@ package gropius.dto.input.architecture
 
 import com.expediagroup.graphql.generator.annotations.GraphQLDescription
 import com.expediagroup.graphql.generator.execution.OptionalInput
+import com.expediagroup.graphql.generator.scalars.ID
 import gropius.dto.input.common.JSONFieldInput
 import gropius.dto.input.common.UpdateNamedNodeInput
 import gropius.dto.input.common.ensureNoDuplicates
+import gropius.dto.input.ensureDistinct
 import gropius.dto.input.ifPresent
 import gropius.dto.input.template.UpdateTemplatedNodeInput
 
 @GraphQLDescription("Input for the updateIMS mutation")
 class UpdateIMSInput(
     @GraphQLDescription("Values for templatedFields to update")
-    override val templatedFields: OptionalInput<List<JSONFieldInput>>
+    override val templatedFields: OptionalInput<List<JSONFieldInput>>,
+    @GraphQLDescription("Ids of permissions to add, must be distinct with removedPermissions.")
+    val addedPermissions: OptionalInput<List<ID>>,
+    @GraphQLDescription(
+        """Ids of permissions to remove, must be distinct with addedPermissions.
+        There must always be at least one permissions granting ADMIN to some GropiusUser left.
+        """
+    )
+    val removedPermissions: OptionalInput<List<ID>>
```

Comment:
-with this many lines of params + descriptions maybe put an empty line before each next param?

Reply:
I could do this  
however it would be inconsistent with the formatting of the rest of the project  
I'm not sure if this is desirable

Reply:
I would propose we do line breaks if the GraaphQLDescription is multiline?

Reply:
tried it, but imho this looks very weird, as then we have empty lines in some places in a constructor, but not in others

Reply:
Hmm. But it looks very crammed and the way it is at the moment it visually looks as if the multiline annotation belongs to the param above as there are two almost empty lines below but it directly connects to the param above

What about a new line btween all params if there is at least one multiline annotation? I think that looks ok, what do you say?

Reply:
I still would prefer a consistent solution  
but we could just move it to the body of the class, and make it a notNull var as I did for the abstract superclasses 
as we never manually call the constructor, I guess this would be an acceptable solution.
However, this would lead to the question, do we want to do this for all params of input classes, or only for those with multiline descriptions? 
In either way, I would do this in a separate PR, as this affects a lot of classes which were not updated in this PR originally

Reply:
Ok, then let's just close this discussion